esp32_ulp/assemble.py

```diff
@@ -277,7 +277,7 @@ def assembler_pass(self, lines):
                     # machine instruction
                     func = getattr(opcodes, 'i_' + opcode.lower(), None)
                     if func is not None:
-                        instruction = func(*args)
+                        instruction = 0 if self.a_pass == 1 else func(*args)
```

Comment:
as this is not easy to understand, maybe copy some text from commit comment into a source comment above this line.

Reply:
Added a description comment